spring-integration-core/src/test/java/org/springframework/integration/config/AggregatorParserTests.java

```diff
@@ -266,5 +302,19 @@ public void testAggregatorFailureIfMutuallyExclusivityPresent() {
 		return MessageBuilder.withPayload(payload).setCorrelationId(correlationId).setSequenceSize(sequenceSize)
 				.setSequenceNumber(sequenceNumber).setReplyChannel(outputChannel).build();
 	}
-	
+
+	public static class MyMGP extends SimpleMessageGroupProcessor {
```

Comment:
Maybe we should keep the new one and deprecate the old if we don't talk about it anywhere.
I prefer the Simple name to PassThrough (it's only used in tests by the framework). But I really don't mind either way.